pytest_checkdocs/__init__.py

```diff
@@ -57,7 +64,15 @@ def system_message(reporter, level, message, *children, **kwargs):
         docutils.utils.Reporter.system_message = orig
 
     def get_long_description(self):
-        return Description.from_md(ensure_clean(pep517.meta.load('.').metadata))
+        builder = build.ProjectBuilder('.')
+        with tempfile.TemporaryDirectory() as tmpdir:
+            try:
+                path = pathlib.Path(builder.prepare('wheel', tmpdir))
+            except TypeError:
+                path = Path(builder.build('wheel', tmpdir))
```

Comment:
Actually, this is not correct, it needs to be joined with the `.dist-info` directory.

Reply:
Fixed now with pypa/build 0.5.0.